_templates/component_v4/with-prompt/defines.ejs.t

```diff
@@ -1,5 +1,5 @@
 ---
-to: <%= path %>/<%= componentName %>/defines.ts
+to: <%= path %>/<%= tagName %>/defines.ts
```

Comment:
Should we also change this to `interfaces.ts` ? Makes more sense to me.

Reply:
Created issue for the topic #74
It being addressed in this PR or in a separate one is irrelevant to me :)
Will approve the PR